src/components/form-field/form-field/form-field.scss

```
@@ -203,15 +213,26 @@
   }
 }
 
+::slotted(*[slot='prefix']) {
```

* should not be needed here
```suggestion
::slotted([slot='prefix']) {
```